packages/next/package.json

```diff
@@ -20,7 +20,7 @@
   "license": "ISC",
   "devDependencies": {
     "@types/express": "^5.0.3",
-    "modelence": "*",
+    "modelence": "^0.6.2",
```

Comment:
Can you revert this change? This will likely create problems with old versions of Modelence (we switched to `*` because of these issues originally)